.github/workflows/publish-runtime.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           GH_WORKFLOW_MATRIX_CHAIN: ${{ matrix.chain }}
           GH_WORKFLOW_MATRIX_SRTOOL_IMAGE: ${{ matrix.srtool_image }}
           GH_WORKFLOW_MATRIX_SRTOOL_IMAGE_TAG: ${{ matrix.srtool_image_tag }}
-          RUNTIME_BUILD_OPTS: "--features on-chain-release-build"
```

Comment:
Was this failing? If yes, RT3200 will not support the metadata hash

Reply:
I've forked from the wrong  base, I'm merging master :+1: